index.html

```diff
@@ -200,13 +206,18 @@ <h2 id="civname">Loading…</h2>
 
     function displayHelp(overlayId) {
         let helptext = document.getElementById("helptext");
+        let helptextContent = document.getElementById("helptext__content");
+        let helptextXRef = document.getElementById("helptext__x_ref");
         let overlay = SVG.get(overlayId);
         let name = overlay.data('name');
         let id = overlay.data('id');
         let caret = overlay.data('caret');
         let type = overlay.data('type');
         let content = getHelpText(name, id, type);
-        helptext.innerHTML = content;
+        helptextContent.innerHTML = content;
+
+        badgeXRef(name, type);
```

Comment:
Can we find a better name for this function? `styleXRefBadges()` or something along those lines?